docs/public/components/shared_features/status.md

```diff
@@ -1,3 +1,180 @@
 # Status
 
-<!-- TODO -->
+Components can register a status provider. When the status command is executed, we will populate the information displayed using all the status providers.
+
+## Status Providers
+
+There are two types of status providers:
+- Header Providers
+- Regular Providers
```

Comment:
I think we should explain the difference here rather than inside the "header provider" section.

```suggestion
- Header Providers: those providers are displayed at the top of the status output. This section is reserved for the most important information about the agent, such as agent version, hostname, host info, metadata, etc.
- Regular Providers: those providers will be rendered after all the header providers,
```